packages/dht/src/helpers/Connectivity.ts

```diff
@@ -3,7 +3,7 @@ import { ConnectivityMethod, NodeType, PeerDescriptor } from '../proto/packages/
 import { isPrivateIPv4 } from './AddressTools'
 
 export const canOpenConnectionFromBrowser = (websocketServer: ConnectivityMethod): boolean => {
-    const hasPrivateAddress = ((websocketServer.host === 'localhost') || isPrivateIPv4(websocketServer.host))
+    const hasPrivateAddress = ((websocketServer.host === '127.0.0.1') || isPrivateIPv4(websocketServer.host))
```

Comment:
Keep this as localhost, it is there intentionally. isPrivateIPv4() will include 127.0.0.1

Reply:
Reverted this change. The `isPrivateIPv4` changed in a separate PR: https://github.com/streamr-dev/network/pull/2239